.travis.yml

```diff
@@ -65,9 +65,11 @@ before_install:
 sudo: required
 after_success:
   - codecov
-  - pip install virtualenv
-  - virtualenv ~/env
-  - source ~/env/bin/activate
-  - pip install transifex-client
-  - sudo echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"$TRANSIFEX_USER"$'\npassword = '"$TRANSIFEX_PASSWORD"$'\ntoken = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc
-  - tx push -s
+  - if [ $TRAVIS_BRANCH == "master" ]; then
```

Comment:
Incase tests on our master branch fail what would we do?

Reply:
You have to follow the same convention, which you are following currently.
(You have to restart test cases manually).

Reply:
Ok.